### ISSUE TITLE
"Missing PDF File" when compiling Latex document from shared network location

### ISSUE THREAD

**Issue by megacorvega** [James-Yu/LaTeX-Workshop#1791]
**Disable all the other extensions except for LaTeX Workshop, and check that you still see this issue. [Required]**

You still see this issue?: Yes

**Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.**

Nothing here relevant, that I can see.

**If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.**

LaTeX PDF compiles fine from copied folder placed on local desktop. PDF file is "missing" when compiled from shared network location on another PC on the same network.

**Describe the bug [Required]**

<details>
<summary>Bug Details</summary>
When trying to view the PDF of a Latex document in Visual Studio Code, this error bubble is shown. The PDF is viewable from Google Chrome using the "View in web browser" option

```
PDF.js v2.2.228 (build: d7afb74a)
Message: Missing PDF
"http://localhost:50604/pdf..JTVDJTVDdGxndy1lbmdzdCU1Q3RsZ3ctZHJpdmUlNUNGQUElNUNGQUElMjBBQ08lNUNGQUElMjBQcm9qZWN0JTIwQ2VydGlmaWNhdGlvbnMlNUNQcm9qZWN0cyU1QzE4MDElMjBGQUElMjBTQTEzMzg3U0UtQSU1Q0RST1AlMjBURVNUJTIwUExBTiU1Q1RQLVRSQS0xODAxLUElNUNJUiU1Q1RQLVRSQS0xODAxLUEucGRm"
```
</details>

**To Reproduce**
1. Open Latex folder location (over shared network folder) from VS Code
2. Open .tex file
3. Attempt to view PDF within VS Code
4. Error is seen

**Expected behavior**
PDF should display in PDF internal window 

**Logs [Required]**
Please paste the whole log messages here, not parts of ones. It is very important to identify problems. If you think the logs are unrelated, please say so.

<details>
<summary>LaTeX Workshop Output [Required]</summary>

I have edited the file path names for privacy/ clarity

```
[13:50:13] Handle data type: close
[13:50:18] VIEW command invoked with mode: file://[network-path]/myfile.tex.
[13:50:18] Found root file from active editor: \\[network-path]\myfile.tex
[13:50:18] Root file remains unchanged from: \\[network-path]\myfile.tex.
[13:50:18] Serving PDF file at http://localhost:50133/viewer.html?file=pdf..JTVDJTVDdGxndy1lbmdzdCU1Q3RsZ3ctZHJpdmUlNUNGQUElNUNGQUElMjBBQ08lNUNGQUElMjBQcm9qZWN0JTIwQ2VydGlmaWNhdGlvbnMlNUNQcm9qZWN0cyU1QzE4MDElMjBGQUElMjBTQTEzMzg3U0UtQSU1Q0RST1AlMjBURVNUJTIwUExBTiU1Q1RQLVRSQS0xODAxLUElNUNJUiU1Q1RQLVRSQS0xODAxLUEucGRm
[13:50:18] The encoded path is \\[network-path]\myfile.pdf
[13:50:18] Open PDF tab for \\[network-path]\myfile.pdf
[13:50:18] Found root file from active editor: \\[network-path]\myfile.tex
[13:50:18] Root file remains unchanged from: \\[network-path]\myfile.tex.
[13:50:19] Error reading PDF file: \[network-path]\myfile.pdf
[13:50:19] Handle data type: open
```

</details>

<details>
<summary>Developer Tools Console [Required]</summary>

This is all the developer tools console outputs

`webviewElement.ts:241 [Embedded Page] Uncaught (in promise) Error: Missing PDF file.`

</details>

**Desktop [Required]:**
 - OS: Windows 10
 - VS Code version: 1.40.0
 - Extension version: 8.3.1

**Are you using LaTeX Workshop with VS Code Remote?**

No

**Comment by jlelong** [comment]
Maybe a duplicate of #1782 
Post the complete extension logs including the `Build` call. Please do not alter the paths in the logs in particular when you suspect that the issue lies somewhere in the paths.

**Comment by tamuratak** [comment]
The cause is that the fix of #1782 is not included in v8.3.1 due to the incorrect release. Related to #1792.

**Comment by megacorvega** [comment]
@jlelong @tamuratak I can also confirm that creating a symlink on the local drive of the network path fixes the issue.
